Library/ViewModels/MessagesViewModel.swift

```diff
@@ -191,12 +198,16 @@ public final class MessagesViewModel: MessagesViewModelType, MessagesViewModelIn
       .ignoreValues()
 
     // TODO: Call blocking GraphQL mutation
-    self.userBlocked = self.blockUserProperty.signal.map { true }
+    self.didBlockUser = self.blockUserProperty.signal.map { false }
 
-    self.userBlocked.observeValues { didBlock in
+    self.didBlockUser.observeValues { didBlock in
       guard didBlock == true else { return }
       NotificationCenter.default.post(.init(name: .ksr_blockedUser))
     }
+
+    /// TODO(MBL-1025): Get isBlocked status from the backend instead.
+    self.participantPreviouslyBlocked = self.viewWillAppearProperty.signal
+      .mapConst(false)
```

Comment:
This is already available in `project.creator.isBlocked` (though not wired up to the backend, so not exactly testable). I'd recommend just using it now; then you can update the test as well.